mongofiles/main/mongofiles.go

```diff
@@ -21,31 +21,38 @@ func main() {
 	opts, err := mongofiles.ParseOptions(os.Args[1:])
 	if err != nil {
 		log.Logv(log.Always, err.Error())
-		os.Exit(util.ExitBadOptions)
+		log.Logv(log.Always, "try 'mongofiles --help' for more information")
```

Comment:
Another example of nearly identical text -- should generate from the program name.  (e.g. `util.ShortUsage("mongofiles")`)

Reply:
Done